data/wanderer/wanderer news.txt

```diff
@@ -0,0 +1,127 @@
+
+news "wanderer civilians, no dialog"
+	location
+		government "Wanderer"
+		not attributes "evacuation"
+	name
+		word
+			"Gardener"
+	message
+		word
+			"You see a Wanderer carefully place some seeds in the ground. They then spread some sort of light grey powder over them."
+			"From the spaceport you can see some buildings with plants growing on their sides. A Wanderer prunes and waters them in mid-air."
+			"While walking through the spaceport, you see a large greenhouse full of various plants. A Wanderer inside is tending to them."
+			
```

Comment:
I'm not sure that, frankly, any of these make a great deal of sense. 
Re:
1) These are their developed worlds, finished even. I doubt very much that the spaceport, the hub of their efforts, is in any way in need of seed planting in its vicinity.
2) Unless the Wanderers can hover with hummingbird like precision (which goes against their existing flight descriptions) _and_ they have hands separate from their wings (as yet unclear I think?) this is just straight up impossible.
3) Greenhouses take up a crapload of space, why would you locate this on ground with the highest demand for logistical considerations?

Reply:
I tried fixing them, are they better?

Reply:
Honestly, not really. I would expect basic watering to be automated for an advanced species with a small population and a lot of ground to cover, and a farm has the same issue as a greenhouse for being next to the spaceport.
you could bend one of them like:
```suggestion
			"Many buildings near the spaceport have plants growing on their roofs. An indignant squawk draws your attention to a Wanderer as they rapidly flap away from an automated watering system dousing them where they stood."
```

Reply:
That's much better, thank you.